extensions/sql-database-projects/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sql-database-projects",
   "displayName": "SQL Database Projects",
   "description": "The SQL Database Projects extension for Azure Data Studio allows users to develop and publish database schemas.",
-  "version": "0.11.0",
+  "version": "0.12.0",
```

Comment:
no changed required for ADS core version?

Reply:
not yet. Might need to bump the ADS dependency if something in data workspace changes before the next release.